css/style.css

```diff
@@ -0,0 +1,253 @@
+.body {
+  position: relative;
+  background: #191765;
+  display: grid;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
```

Comment:
- Kindly add to buttons is `` <a> `` link a or button tag, and follow the button styling: Active, Hover and Pressed.